active_learning/learning.py

```diff
@@ -31,7 +54,21 @@ def __init__(self, **kwargs):
             self.X.append(article.text)
             self.y.append(article.class_label_id)
 
+
     def classify_svm(self, X, y):
+        '''
+        Uses SVM to classify articles before calculating and printing the accuracy
+
+        Parameters
+        ----------
+        self : self reference
```

Comment:
remove this, as we don't actually pass a self reference when calling this function